src/views/users/User/UserList.vue

```diff
@@ -65,7 +67,16 @@ export default {
             }
           },
           mfa_level: {
-            width: '130px'
+            width: '130px',
+            formatter: (row) => {
+              if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.allUsers) {
+                return this.$t('MFAAllUsers')
+              } else if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === MFASystemSetting.onlyAdminUsers && (row?.is_superuser || row?.is_org_admin)) {
+                return this.$t('MFAOnlyAdminUsers')
+              } else {
+                return row['mfa_level'].label
+              }
+            }
           },
           source: {
             width: '120px',
```

Comment:
There appears to be some syntax issues at line 65 onwards, which may affect proper execution of the code. Additionally, there is no actual data being passed into the `component` function that initializes components, so the first component should probably contain a `<router-view>` since it's supposed to render based on route. The other parts appear fine with minor improvements such as ensuring that all functions/methods in imports are properly utilized.